subcommands/common_config.go

```diff
@@ -235,6 +235,16 @@ func SetUpdatesConfig(opts *SetUpdatesConfigOptions, reportedTag string, reporte
 		fmt.Printf("Setting apps to [%s]\n", opts.UpdateApps)
 		sota.Set("pacman.docker_apps", opts.UpdateApps)
 		sota.Set("pacman.compose_apps", opts.UpdateApps)
+		if strings.TrimSpace(opts.UpdateApps) == "-" {
+			err = sota.Delete("pacman.docker_apps")
+			if err != nil {
+				fmt.Printf("Error trying to delete pacman.docker_apps key: %s\n", err)
+			}
+			sota.Delete("pacman.compose_apps")
+			if err != nil {
+				fmt.Printf("Error trying to delete pacman.compose_apps key: %s\n", err)
+			}
```

Comment:
Looking at `sota.Delete` IIUC there are two use cases when it can return an error:
1. A key is malformed.
2. A key is not found.

Here, (1) is a programming error deserving `DieNotNil`, while (2) would be better off silently ignored.
Thus, I suggest to improve this code using `sota.Has`.

```suggestion
if sota.Has("pacman.docker_apps") {
    subcommands.DieNotNil(sota.Delete("pacman.docker_apps"))
}
if sota.Has("pacman.compose_apps") {
    subcommands.DieNotNil(sota.Delete("pacman.compose_apps"))
}
```

Reply:
The other option, as we know a key we pass is properly formatted, would be to make an "optimistic" assumption that `sota.Delete` always succeeds from a practical perspective:
```suggestion
			_ = sota.Delete("packman.docker_apps")
			_ = sota.Delete("packman.compose_apps")
```

Reply:
Even though we have properly formatted ones, maybe in the future some other error check will be introduced so I went with the aforementioned one. Check of the key and delete it.